src/schemas/Arranger/initArrangerMetadata.ts

```diff
@@ -0,0 +1,80 @@
+import { Client } from '@elastic/elasticsearch';
+import isEqual from 'lodash/isEqual';
+import retry from 'async-retry';
+import { ARRANGER_PROJECT_ID, ARRANGER_FILE_CENTRIC_INDEX } from 'config';
+import logger from 'utils/logger';
+import metadata from 'resources/arranger_es_metadata.json';
+
+export const ARRANGER_PROJECT_METADATA_INDEX = `arranger-projects-${ARRANGER_PROJECT_ID}`;
+export const ARRANGER_PROJECTS_INDEX = `arranger-projects`;
+
+export const harmonizedFileCentricConfig: typeof metadata.projectIndexConfigs.file_centric = {
+  ...metadata.projectIndexConfigs.file_centric,
+  index: ARRANGER_FILE_CENTRIC_INDEX || metadata.projectIndexConfigs.file_centric.index,
```

Comment:
I think we should use ARRANGER_FILE_CENTRIC_INDEX only. It could get confusing trying to figure out if gateway's arranger is using the env var or the one defined in the `resource/...meta.json`. Like the test is also using ARRANGER_FILE_CENTRIC_INDEX

Reply:
yeah I guess we can provide a consistent default in the config file so local dev doesn't need this set :) will update!